keyboards/nacly/sodium50/info.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Please fix the ordering. Press Ctrl+Shift+I in the Configurator and turn on Fast Input to verify.
![image](https://user-images.githubusercontent.com/4781841/169741115-92c73dfc-03e3-484b-8a0a-45f27cf0223a.png)

Reply:
Is there a good way to fix the ordering? I copied the layout from kle and used the conversion tool. Also is there a way to view my keyboards in the configurator before it's committed?

Reply:
Not sure i'm following. ctrl + shift + i on mine doesn't do anything.

Reply:
Make sure you don't have anything selected.

Reply:
That was just to illustrate that the ordering is incorrect.
> Press Ctrl+Shift+I in the Configurator and turn on Fast Input to verify.

Reply:
Sorry this is commented on the 50, but the screenshot is of the 62

Reply:
Doesn't matter, the process is the same for any board.

Reply:
Ok i'm seeing it now what you mean with the order is wrong. What controls the order? The list in the json, or the x and y of the key?

Reply:
So if I make it so it's a square matrix instead of representing the columnar stagger it works in order. How can I keep the columnar stagger and keep the order correct
<img width="967" alt="Screen Shot 2022-05-25 at 11 21 13 AM" src="https://user-images.githubusercontent.com/4699432/170298506-5eb4f15f-f5a2-4351-a0c2-c30bc7e2d934.png">
?

Reply:
You don't need to change the physical position. Just change the ordering in the JSON.

Reply:
Sorry for the delay. The ordering should be fixed.